pisek/utils/user_input.py

```diff
@@ -40,3 +43,21 @@ def input_choice(message: str, choices: Sequence[tuple[T, str]]) -> T:
             selected = (selected - 1) % len(choices)
 
         print(Cursor.UP() * len(choices), end="")
+
+
+def input_choice_no_jumps(message: str, choices: Sequence[tuple[T, str]]) -> T:
+    assert choices
+
+    print(message)
+
+    for i, (_, text) in enumerate(choices):
+        full_text = f" {i+1}. {text}"
+        print(" " + full_text)
+
+    while True:
+        try:
+            selected = int(input(f"Select number from 1 to {len(choices)}: ")) - 1
```

Comment:
With `no_jumps`, you're not selecting a number, you're entering it (to select an option).

```suggestion
            selected = Enter number from 1 to {len(choices)}: ")) - 1
```